test/parsing-test.js

```diff
@@ -21,49 +22,35 @@ describe('Messages parsing', function () {
     })
 
     it('Action getting', function () {
-        assert.equal(messages.getAction(msg), "+todo")
+        assert.equal(formatting.getDataMask(msg, /\+[^*\s]+/), "+todo")
```

Comment:
If in command typo mistake it won't be executed at all on the stage where we are looking for action